<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>286</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="DataSet_NewDataSet_button">
         <property name="toolTip">
          <string>Create new DataSet</string>
         </property>
         <property name="statusTip">
          <string>Create new DataSet</string>
         </property>
         <property name="text">
          <string>&amp;New Data Set</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DataSet_DeleteDataSet_button">
         <property name="toolTip">
          <string>Delete selected DataSet</string>
         </property>
         <property name="statusTip">
          <string>Delete selected DataSet</string>
         </property>
         <property name="text">
          <string>Delete Data Set</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="DataSet_DataSets_listView">
         <property name="toolTip">
          <string>List of DataSets</string>
         </property>
         <property name="statusTip">
          <string>List of DataSets</string>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::NoDragDrop</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::CopyAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="DataSet_AddFiles_button">
         <property name="toolTip">
          <string>Add DataFile(s)</string>
         </property>
         <property name="statusTip">
          <string>Add DataFile(s)</string>
         </property>
         <property name="text">
          <string>Add &amp;Files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DataSet_DeleteFiles_button">
         <property name="toolTip">
          <string>Delete selected DataFile(s)</string>
         </property>
         <property name="statusTip">
          <string>Delete selected DataFile(s)</string>
         </property>
         <property name="text">
          <string>Delete Files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="DataSet_filenames_listView">
         <property name="toolTip">
          <string>List of DataFIle(s) in selected DataSet</string>
         </property>
         <property name="statusTip">
          <string>List of DataFIle(s) in selected DataSet</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::NoDragDrop</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="toolTip">
            <string>Select binning for conversion of DataSet</string>
           </property>
           <property name="statusTip">
            <string>Select binning for conversion of DataSet</string>
           </property>
           <property name="text">
            <string>Select Binning</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="DataSet_binning_comboBox">
           <property name="toolTip">
            <string>Select binning for conversion of DataSet</string>
           </property>
           <property name="statusTip">
            <string>Select binning for conversion of DataSet</string>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DataSet_convertData_button">
           <property name="toolTip">
            <string>Convert selected DataSet</string>
           </property>
           <property name="statusTip">
            <string>Convert selected DataSet</string>
           </property>
           <property name="text">
            <string>Convert</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="DataSet_fileAttributs_listView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>List of properties of selected DataFile(s)</string>
         </property>
         <property name="statusTip">
          <string>List of properties of selected DataFile(s)</string>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DropOnly</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::IgnoreAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
